samples/msal-node-samples/ElectronReactTestApp/src/app/App.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import React from 'react';
+import { IpcMessages } from '../Constants';
+import PageLayout from './components/PageLayout';
+import { Profile } from './pages/Profile';
+import './styles/App.css';
+
+//The ipcRenderer module is an EventEmitter. It provides a few methods so you can send synchronous and asynchronous messages
+const { ipcRenderer } = window.require('electron');
+
+const Pages = () => {
+    return (
+        <Routes>
+            <Route path="/profile" element={<Profile />} />
```

Comment:
I would add a home page here as placeholder to eliminate the `No routes matched location "/" `warning from react-router

Reply:
Done!